src/SurveyService.Models.Dto/Requests/Question/CreateQuestionRequest.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+
+public record CreateQuestionRequest
+{
+  public string Content { get; set; }
```

Comment:
requred attribute

Reply:
I'd better just remove CreateQuetionRequest, this model someone else's responsibility.